### PR TITLE
Freeze, unfreeze and close deposit accounts

### PR DESCRIPTION
**Pull request by sanjani-thatch**
Want to be able to freeze, unfreeze and close unit deposit accounts

### CODE REVIEW

**Comment by sanjani-thatch** [comment]
Testing included in thatch repo (using stubs)

File: lib/unit-ruby/deposit_account.rb [hunk 1]
```diff
@@ -24,5 +24,51 @@ class DepositAccount < APIResource
     include ResourceOperations::Create
     include ResourceOperations::Save
     include ResourceOperations::Find
+
+    def resource_path
+      self.class.resource_path(id)
+    end
+
+    def close (reason)
```
Comment (bdewater-thatch) [inline lib/unit-ruby/deposit_account.rb]: Nit: usually there's no space between the method name and arguments when using parenthesis. Ruby also allows `def close reason` but that's not generally the style we follow.

```suggestion
    def close(reason)
```

File: lib/unit-ruby/deposit_account.rb [hunk 2]
```diff
@@ -24,5 +24,51 @@ class DepositAccount < APIResource
...
+              reason: reason,
+            }.compact
+          }
+        }
+      )
+
+      update_resource_from_json_api(updated_resource)
+    end
+
+    def freeze (reason, reasonText)
```
Comment (bdewater-thatch) [inline lib/unit-ruby/deposit_account.rb]: Same spacing nit + using snake_case instead of camelCase:
```suggestion
    def freeze(reason, reason_text)
```

**Review by bdewater-thatch — COMMENTED** [review]
Couple of stylistic nits but otherwise it looks good to go once those are merged. 

We should either setup Rubocop here or just vendor the gem in the app at this point, but that's another issue altogether :)

**Pull request merged**